rero_ils/modules/documents/extensions/contributions.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2021 RERO
+# Copyright (C) 2021 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Document record extension to enrich the edition statement."""
+
+
+from invenio_records.extensions import RecordExtension
+
+from rero_ils.dojson.utils import remove_trailing_punctuation
+
+from ..utils import display_alternate_graphic_first
+
+
+class EditionStatementExtension(RecordExtension):
+    """Adds textual information for editionStatement."""
+
+    @classmethod
```

Comment:
really a class method ? I didn't notice any `cls` usage.
It looks like a `staticmethod` for me (and a private one)